components/nav/NavTitle.vue

```diff
@@ -18,20 +18,17 @@ router.afterEach(() => {
 </script>
 
 <template>
-  <div flex justify-between sticky top-0 bg-base z-1 py-4 native:py-7 data-tauri-drag-region>
+  <div flex sm-justify-center xl-justify-between sticky top-0 bg-base z-1 py-4 native:py-7 data-tauri-drag-region>
```

Comment:
the mobile-view version of the Home logo will receive `justify-content: center` to match the other logos below in the column (the text next to each logo gets hidden in this state)